build/config/PublishData.json

```diff
@@ -35,6 +35,13 @@
             "vsix": [ "https://dotnet.myget.org/F/roslyn/vsix/upload" ],
             "channels": [ "dev16.0p1" ]
         },
+        "dev16.0-preview2-vs-deps": {
+            "nugetKind": ["Shipping", "NonShipping"],
+            "version": "2.11.*",
+            "nuget": [ "https://dotnet.myget.org/F/roslyn/api/v2/package" ],
+            "vsix": [ "https://dotnet.myget.org/F/roslyn/vsix/upload" ],
+            "channels": [ "dev16.0p2" ]
```

Comment:
You need to update the master-vs-deps one now to say dev16.0p3.

Reply:
Yes, thanks

Reply:
@jasonmalinowski Look good now?